 * Serving Flask app 'app'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:5000
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 373-924-250
127.0.0.1 - - [10/May/2025 01:10:59] "GET / HTTP/1.1" 404 -
127.0.0.1 - - [10/May/2025 01:11:02] "GET /students HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/codespace/.local/lib/python3.12/site-packages/flask/app.py", line 1536, in __call__
    return self.wsgi_app(environ, start_response)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/flask/app.py", line 1514, in wsgi_app
    response = self.handle_exception(e)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/flask/app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/flask/app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/flask/app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/codespace/.local/lib/python3.12/site-packages/flask/app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspaces/Servicio-web-en-py/app.py", line 15, in get_all_students
    students = conn.execute('select * from students').fetchall()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such table: students
 * Detected change in '/workspaces/Servicio-web-en-py/models.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 373-924-250
 * Detected change in '/workspaces/Servicio-web-en-py/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 373-924-250
 * Detected change in '/workspaces/Servicio-web-en-py/app.py', reloading
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 373-924-250
